addons/barcodes_gs1_nomenclature/models/barcode_nomenclature.py

```diff
@@ -139,7 +139,7 @@ def _preprocess_gs1_search_args(self, args, barcode_types, field='barcode'):
         is only digits to keep the original barcode part only.
         """
         nomenclature = self.env.company.nomenclature_id
-        if nomenclature.is_gs1_nomenclature:
+        if nomenclature.is_gs1_nomenclature and not self.env.context.get('skip_gs1_pre', False):
```

Comment:
```suggestion
        if nomenclature.is_gs1_nomenclature and not self.env.context.get('skip_gs1_pre'):
```
No need to add a default value of `False`, as `None` has the same boolean evaluation.

Reply:
removed the default